weblate/templates/snippets/project-nav.html

```diff
@@ -0,0 +1,143 @@
+{% load i18n %}
+{% load translations %}
+
+{% url 'show_dictionaries' project=object.slug as show_dictionaries %}
+
+<ul class="nav nav-pills">
+    {% if request.path == show_dictionaries %}
```

Comment:
It would be probably better to compare against project URL here to make the code prepared for inclusion in another pages. I wonder if we could make it without these conditions...

Reply:
Not sure how to tackle this one without major changes, since a different view is taking care of the `/dictionaries/project/` urls. I could try to improve the templates perhaps, but the `data-toggle="tab"` is a pain.

Any suggestions on how to proceed?

Reply:
I think what you did is good enough for now. In case you have further ideas to improve, please do that :-).